src/OmniSharp.MSBuild/ProjectFile/PropertyConverter.cs

```diff
@@ -119,9 +119,7 @@ public static NullableContextOptions ToNullableContextOptions(string propertyVal
             {
                 case "disable": return NullableContextOptions.Disable;
                 case "enable": return NullableContextOptions.Enable;
-                case "safeonly": return NullableContextOptions.SafeOnly;
```

Comment:
ooh I didn't realize LDM decided to drop this 🧐 https://github.com/dotnet/roslyn/issues/35408